### PR TITLE
Doc: How to use `built-in` dataset

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by TheaperDeng** [comment]
The notebook looks good. Here are some comments:
1. We could remove the paragraph of "Description of parameters" and just leave our API doc link.
2. add a special note for tsinghua_electricity becasue we ask users to download it manually.

Please rebase the PR and generate a doc link here.

**Comment by liangs6212** [comment]
> The notebook looks good. Here are some comments:
> 
> 1. We could remove the paragraph of "Description of parameters" and just leave our API doc link.
> 2. add a special note for tsinghua_electricity becasue we ask users to download it manually.
> 
> Please rebase the PR and generate a doc link here.

Done, doc link: https://liangs.readthedocs.io/en/doc-how-to-built-in-dataset/doc/Chronos/Howto/how_to_use_built-in_datasets.html

**Comment by TheaperDeng** [comment]
sorry i miss this PR, could you rebase it again?

File: docs/readthedocs/source/doc/Chronos/Howto/index.rst [hunk 1]
```diff
@@ -24,6 +24,7 @@ Forecasting
 * `Export the OpenVINO model files to disk <how_to_export_openvino_files.html>`__
 * `Export the TorchScript model files to disk <how_to_export_torchscript_files.html>`__
 * `Preprocess my data <how_to_preprocess_my_data.html>`__
+* `Built-in dataset <how_to_use_built-in_datasets.html>`__
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/Howto/index.rst:27]: put this to data section as well

**Pull request merged**